src/Api/Order.php

```diff
@@ -14,6 +15,11 @@
 class Order extends AbstractEndpoint
 {
 
+	const SHIPPING_REQUIRED_FIELDS = [ 'first_name', 'last_name', 'address_1', 'city', 'state', 'postcode', 'country', 'email' ];
```

Comment:
What happen if we change the required fields from a woocomerce hook like: 

- `woocommerce_billing_fields`